content/client-lib-development-guide/features.textile

```diff
@@ -274,10 +276,11 @@ h3(#realtime-connection). Connection
 ** @(RTN2b)@ @echo@ should be @true@ by default; @false@ will prevent messages published by the client being echoed back
 ** @(RTN2d)@ @clientId@ contains the provided @clientId@ option of @ClientOptions@, unless @clientId@ is @null@
 ** @(RTN2e)@ Depending on the authentication scheme, either @accessToken@ contains the token string, or @key@ contains the API key
-** @(RTN2f)@ API version param @v@ should be @0.8@
+** @(RTN2f)@ API version param @v@ should be @0.9@
 * @(RTN3)@ If connection option @autoConnect@ is true, a connection is initiated immediately; otherwise a connection is only initiated following an explicit call to @connect()@
 * @(RTN4)@ EventEmitter and states:
 ** @(RTN4a)@ Implements @EventEmitter@ and emits events for state changes @INITIALIZED@, @CONNECTING@, @CONNECTED@, @DISCONNECTED@, @SUSPENDED@, @CLOSING@, @CLOSED@, @FAILED@
+** @(RTN4g)@ Additionally, an @ERROR@ event is emitted that contains an @ErrorInfo@ object with details on an error that has occurred for the @Connection@
```

Comment:
Is this intended just for (eg resume) errors on `CONNECTED`? Or all errors (including fatal ones in `ERROR`s and nonfatal in `DISCONNECTED`s), duplicating the ErrorInfo in `connectionStateChange.reason`?

If just for `CONNECTED` errors, does that mean that those shouldn't be emitted in the `connectionStateChange.reason`? (I guess it would make sense for them not to be, since a resume failure isn't the reason why you're connected, but if so that should be made explicit in RTN4f)